sources/packages/backend/apps/workers/src/main.ts

```diff
@@ -4,9 +4,10 @@ import { WorkersModule } from "./workers.module";
 import { ZeebeTransportStrategy } from "./zeebe/zeebe-transport-strategy";
 
 async function bootstrap() {
-  const workers = await NestFactory.createMicroservice(WorkersModule, {
-    strategy: new ZeebeTransportStrategy(),
+  const workers = await NestFactory.create(WorkersModule);
+  workers.connectMicroservice({
+    strategy: workers.get(ZeebeTransportStrategy),
   });
-  await workers.listen();
+  await workers.startAllMicroservices();
```

Comment:
Very curious to know what is the difference here. But I can read the documentation as well.

Reply:
The key point is about retrieving from the injector the `ZeebeTransportStrategy` instead of creating a `new ZeebeTransportStrategy()` as was done before.

![image](https://user-images.githubusercontent.com/61259237/196827824-487cb7c4-36a8-4ef3-8583-95c8c299ea75.png)

The main intention was to allow the `ZBClient` to be shared and injected also into the `ZeebeTransportStrategy`, instead of creating a new specific instance to be consumed only into the `ZeebeTransportStrategy`.

![image](https://user-images.githubusercontent.com/61259237/196827979-1053eec1-f114-4e49-b0e0-41a4dfd5f250.png)